### PR TITLE
add error message when trying to uninstall Xcode if file not found

### LINKED ISSUE #144: Uninstall command should give error message when it can't find version to uninstall
**Describe the bug**
I am trying to uninstall a version of Xcode by right-clicking, Uninstall but it does not uninstall.

**To Reproduce**
1. Launch Xcodes
2. Right click on an installed Xcode
3. Hit Uninstall

**Expected behavior**
That version of Xcode should be removed. It is not removed.

**Screenshots**
https://user-images.githubusercontent.com/385415/127566519-ab2c338b-4782-40a7-a258-780789086ef3.mov

**Version**
 - OS: 11.5
 - Xcodes: 1.1.0 (7)

**Console Log**
This is from when I right click and hit Uninstall:
````
default	14:11:15.235008-0700	runningboardd	Invalidating assertion 215-129-13762 (target:[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193]) from originator [daemon<com.apple.coreservices.launchservicesd>:129]
default	14:11:15.247926-0700	runningboardd	Assertion did invalidate due to timeout: 215-129-13764 (target:[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193])
default	14:11:15.248198-0700	runningboardd	Invalidating assertion 215-179-13763 (target:[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193]) from originator [daemon<com.apple.WindowServer(88)>:179]
default	14:11:15.353323-0700	runningboardd	[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193] Ignoring jetsam update because this process is not memory-managed
default	14:11:15.353329-0700	runningboardd	Calculated state for app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>: running-active (role: UserInteractiveNonFocal)
default	14:11:15.353343-0700	runningboardd	[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193] Ignoring suspend because this process is not lifecycle managed
default	14:11:15.353383-0700	runningboardd	[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193] Set darwin role to: UserInteractiveNonFocal
default	14:11:15.353404-0700	runningboardd	[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193] Ignoring GPU update because this process is not GPU managed
default	14:11:15.353600-0700	runningboardd	[app<application.com.robotsandpencils.XcodesApp.96361176.96361182(780288435)>:35193] Skipping AppNap state - not lifecycle managed
````

### PR DESCRIPTION
**Pull request by leon-wolf**
resolves #144:

- check if file exists, which the user is trying to delete
- throw FileNotFound Error when file is not found. Localized in German and English for now.

### CODE REVIEW

File: Xcodes/Backend/FileError.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//
+//  FileError.swift
+//  Xcodes
+//
+//  Created by Leon Wolf on 06.10.22.
+//  Copyright © 2022 Robots and Pencils. All rights reserved.
+//
+
+import Foundation
```
Comment (MattKiazyk) [inline Xcodes/Backend/FileError.swift:9]: Same as your other PR - this needs to be added to the project.

**Review by MattKiazyk — CHANGES_REQUESTED** [review]
Nice clean change  - thanks!

Just fix up adding it into the project

**Comment by MattKiazyk** [comment]
@leon-wolf sorry with the last merge there's a conflict with new files. When you have a moment can you fix it up?

**Comment by leon-wolf** [comment]
> @leon-wolf sorry with the last merge there's a conflict with new files. When you have a moment can you fix it up?

yeah sure. no problem :)

**Review by MattKiazyk — APPROVED** [review]

**Pull request merged**